pkg/ocm/helpers.go

```diff
@@ -869,6 +870,9 @@ func ValidateOperatorRolesMatchOidcProvider(reporter *reporter.Object, awsClient
 			return err
 		}
 		roleARN := *roleObject.Arn
+		if printInfo {
+			reporter.Infof("Using '%s'", *roleObject.Arn)
```

Comment:
I think you moved this too far up. This print should only happen after it passes some validations.
I am thinking maybe before the `if hasManagedPolicies {` check.